src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+
+  font-family: $main-font;
+  line-height: $main-line-height;
+  font-weight: $main-font-weigh;
```

Comment:
There is a typo in the variable name `$main-font-weigh`. It should be `$main-font-weight` as defined in your variables file.